### PR TITLE
Add rating module.

### PR DESCRIPTION
**Pull request by EmilyArkade**

### CODE REVIEW

**Comment by EmilyArkade** [comment]
[Migrate Zendesk plugin to use Ratings endpoint](https://trello.com/c/stadC7fC/182-migrate-zendesk-plugin-to-use-ratings-endpoint)

File: src/Modules/Ratings.php [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        return $this->buildPaginatedCollection(
+            $this->client->get('ratings', $this->applyConstraint($constraint)),
+            Rating::class,
+            [$this, __FUNCTION__],
+            $constraint
+        );
+    }
+
+    /**
+     * Fetch a single ratings.
```
Comment (drewsymo) [inline src/Modules/Ratings.php]: single rating *

File: src/Modules/Ratings.php [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+     */
+    public function delete(Rating $rating)
+    {
+        if (! $rating->id) {
+            throw new \DomainException('Rating must contain an ID to delete');
+        }
+
+        $this->client->delete(sprintf('ratings/%d', $rating->id));
+    }
+}
```
Comment (drewsymo) [inline src/Modules/Ratings.php]: New line

File: src/Modules/Ratings.php (hunk 1 above)
Comment (EmilyArkade) [reply src/Modules/Ratings.php]: Fixed~

File: src/Modules/Ratings.php (hunk 2 above)
Comment (EmilyArkade) [reply src/Modules/Ratings.php]: Fixed~

**Pull request merged**